### PR TITLE
Reset miniscreen after pi-topd is initialized

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/OS-1379) |

#### Main changes

Add new `PUB_PITOPD_READY` message to be handled by `pi-topd` when it's initialized.

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: tests/test_oled.py [hunk 1]
```diff
@@ -275,3 +275,27 @@ def test_display_multiline_text_with_newlines(oled, snapshot, fonts_mock):
     text = "Line1\nLine2\nLine3\nLine4"
     oled.display_multiline_text(text)
     snapshot.assert_match(to_bytes(oled.image), "defaults.png")
+
+
+def test_subscribes_to_pitopd_ready_event(oled):
+    from pitop.common.ptdm import Message
+
+    callback = oled._ptdm_subscribe_client._callback_funcs.get(Message.PUB_PITOPD_READY)
```
Comment (angusjfw) [inline tests/test_oled.py:283]: it might be best to keep this as `__callback_funcs` and use the ._Class__callback_funcs trick to access it in the test, rather than change the actual code to make testing simpler

**Review by angusjfw — APPROVED** [review]
